api/src/controllers/changes.js

```diff
@@ -299,9 +296,10 @@ const initFeed = (req, res) => {
       return filterPurgedIds(feed);
     })
     .then(() => {
-      if (feed.allowedDocIds.length > usersService.DOC_IDS_WARN_LIMIT) {
-        docCountUserWarnings[feed.req.userCtx.name] = feed.allowedDocIds.length;
+      if (feed.req.userCtx && feed.req.userCtx.name) {
+        replicationLimitLog.put(feed.req.userCtx.name, feed.allowedDocIds.length, usersService.DOC_IDS_WARN_LIMIT);
```

Comment:
The third param here is no longer used by the service, so this should just be

```suggestion
        replicationLimitLog.put(feed.req.userCtx.name, feed.allowedDocIds.length);
```

Reply:
Fixed